tests/h/session_test.py

```diff
@@ -5,7 +5,6 @@
 from h.services.list_groups import ListGroupsService
 
 
-@pytest.mark.usefixtures('scope_feature_off')
```

Comment:
Do the `TestModel` tests and `TestModelWithScopedGrouped` tests now need to be merged into one fixture?

Reply:
Unless I misunderstand, this has already been done!